### PR TITLE
feat(cxl-ui): refactor cxl-dashboard-header

### PR DESCRIPTION
**Pull request by HenerHoop**
https://app.clickup.com/t/86ayhhfgy

Slider is currently using the old card until the light-card update gets merged.

### CODE REVIEW

File: packages/cxl-ui/src/components/cxl-dashboard-header.js [hunk 1]
```diff
@@ -1,132 +1,90 @@
...
+  @property({ type: String }) subtitle = 'My dashboard';
 
-  @property({ type: String }) cta1Link = '';
+  @property({ type: String }) title = 'Hello';
 
-  @property({ type: String }) cta2 = '';
+  @property({ type: String }) name = '';
 
-  @property({ type: String }) cta2Link = '';
+  @property({ type: Boolean }) hasCompletedStats = false;
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-dashboard-header.js:29]: I think `show` is better in the context of (not)showing a piece of component.

```suggestion
  @property({ type: Boolean }) showCompletedStats = false;
```

File: packages/cxl-ui/src/components/cxl-dashboard-header.js [hunk 2]
```diff
@@ -1,132 +1,90 @@
...
 
-  @property({ type: String }) cta3 = '';
+  @property({ type: Boolean }) hasContinueSlider = false;
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-dashboard-header.js:31]:
```suggestion
  @property({ type: Boolean }) showContinueSlider = false;
```

**Review by pawelkmpt — CHANGES_REQUESTED** [review]

File: packages/cxl-lumo-styles/scss/global.scss [hunk 3]
```diff
@@ -8,6 +8,7 @@ html {
   --cxl-wrap-padding: var(--lumo-space-m);
   --cxl-space-sm: 12px;
   --cxl-color-light-gray: hsla(0, 0%, 96%, 1);
+  --cxl-color-athens-gray: hsla(210, 20%, 96%, 1);
```
Comment (freudFlintstone) [inline packages/cxl-lumo-styles/scss/global.scss:11]: Maybe not to action now, but I think these are being added to the wrong place. There's a colors.scss file, in which we should use theme lumo variables when possible, but should also have the extra variables.

**Review by freudFlintstone — APPROVED** [review]
Nice work

**Comment by HenerHoop** [comment]
@pawelkmpt I think that this PR is no longer valid because the roadmap block's PR already includes it (with small changes). It was necessary to add it there to see the entire header together.

**Pull request closed without merging**